console/console-init/ui/src/modules/iot-device/components/CreateMetadata/CreateMetadata.tsx

```diff
@@ -45,34 +47,43 @@ export const CreateMetadata: React.FC<ICreateMetadataProps> = ({
     setMetadataList([
       ...metadataList,
       {
```

Comment:
you have already function `getInitialMetadataState` for intial  Metadata state, call that function here.